.eslintrc

```diff
@@ -1,3 +1,24 @@
 {
-  "extends": "rackt"
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "rules": {
+    "array-bracket-spacing": [ 2, "always" ],
+    "comma-dangle": [ 2, "never" ],
```

Comment:
This one is divisive – I used to hate it, but it does make diffs potentially be cleaner.